### PR TITLE
Ichain redirection

### LINKED ISSUE #1054: Register with Ichain: redirect to login page
https://github.com/openSUSE/osem/blob/master/app/controllers/conference_registrations_controller.rb#L17

It is not helpful if it takes you back to root_path. It should take you in login page.

### PR DESCRIPTION
**Pull request by arun1595**
Fixes #1054

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -45,14 +45,20 @@ def not_signed_in
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can [:show, :create], User
-    unless ENV['OSEM_ICHAIN_ENABLED'] == 'true'
+    if ENV['OSEM_ICHAIN_ENABLED'] == 'true'
+      can [:new, :create], Registration do |registration|
```
Comment (differentreality) [inline app/models/ability.rb]: That's not accurate. This is authorization rules for a not_signed_in user. 
If ICHAIN is enabled and the user is **not** logged in, then the user **cannot** access new and create actions for Registration.

Comment (arun1595) [inline app/models/ability.rb]: I am missing something here. In the controller method, we're redirecting the user to ichain login page when he tries to create a new registration. So, the user can only  create a new registration after he's signed in right?

Or am I wrong?

Comment (differentreality) [inline app/models/ability.rb]: That's right

**Comment by differentreality** [comment]
Closing due to inactivity. There is also another PR about this now.

**Pull request closed without merging**